draft-ietf-mls-protocol.md

```diff
@@ -549,6 +537,62 @@ X, then the tree will have the following structure.
 X    B          C    D
 ~~~~~
 
+A node secret is derived from its path secret. The path secret
```

Comment:
I would move this back up to where you deleted a chunk. Instead of "The contents of a parent node...", you should say something like "The tree is always updated along a direct path from a leaf to the root... [path secrets, node secrets]"